dashboard/modules/log/log_manager.py

```diff
@@ -126,7 +126,7 @@ async def stream_logs(
             yield streamed_log.data
 
     def _verify_node_registered(self, node_id: str):
-        if node_id not in self.client.get_all_registered_agent_ids():
+        if node_id not in self.client.get_all_registered_runtime_env_agent_ids():
```

Comment:
I think we shouldn't change this line (this is looking for the agent to fetch logs,' not runtime env?)

Reply:
Updated the code by adding `get_all_registered_log_agent_ids` to state_manager.py to make it more explicit.